src/ripple/protocol/impl/Feature.cpp

```diff
@@ -456,6 +456,7 @@ REGISTER_FIX    (fixReducedOffersV1,            Supported::yes, VoteBehavior::De
 REGISTER_FEATURE(Clawback,                      Supported::yes, VoteBehavior::DefaultNo);
 REGISTER_FEATURE(AMM,                           Supported::yes, VoteBehavior::DefaultNo);
 REGISTER_FEATURE(XChainBridge,                  Supported::yes, VoteBehavior::DefaultNo);
+REGISTER_FIX    (fixPayChanV1,                  Supported::yes, VoteBehavior::DefaultNo);
```

Comment:
Shouldn't this be Yes by default? Per my understanding, every fix amendment shout have vote by default set to yes.

```
REGISTER_FEATURE(fixPayChanV1, Supported::yes, VoteBehavior::DefaultYes);
```

Reply:
Look above at all the fixes that don't have the `VoteBehavior::DefaultYes`.

`Yes` is reserved for things that need immediate approval. This doesn't need immediate approval imo.

I'm happy to change it though